ziti-tunnel/docker/BUILD.md

```diff
@@ -0,0 +1,91 @@
+The Dockerfile and scripts in this directory build a ziti-tunnel Docker image.
+
+Ziti binaries are downloaded from https://netfoundry.artifactory.io/netfoundry/ziti-release
+by default. The following build arguments are supported:
+
+  | Build Argument       | Description                                                                 |
+  | -------------------- | ----------------------------------------------------------------- |
+  | ZITI_VERSION         | The version of the ziti binaries to fetch from artifactory.       |
+  | ARTIFACTORY_TOKEN    | An Artifactory token with read access to the artifact repository. |
```

Comment:
should we make an issue that will expose the repo so this isn't needed? (and subsequently remove from here)

Reply:
The ziti-release repo is now open, but the token argument remains for when we want to pull from other repos (e.g. ziti-snapshots), or if someone wanted to pull from their own private repo.